packages/charts/src/components/ColumnChartWithTrend/ColumnChartWithTrend.tsx

```diff
@@ -183,7 +186,7 @@ const ColumnChartWithTrend: FC<ColumnChartWithTrendProps> = forwardRef(
           onClick={onClick}
           syncId={'trend'}
           ChartPlaceholder={ChartPlaceholder ?? ColumnChartWithTrendPlaceholder}
-          dataset={dataset}
+          dataset={columnDataset}
```

Comment:
why do we need the columnDataset here? I don't see a difference in the chart expect for an incorrect tooltip?
![image](https://user-images.githubusercontent.com/8775211/130398196-35cde34a-6db9-4e80-97a5-cf796b56156b.png)